lib/config/definitions.js

```diff
@@ -1150,6 +1150,17 @@ const options = [
     mergeable: true,
     cli: false,
   },
+  {
+    name: 'gradle',
+    description: 'Configuration object for build.gradle files',
+    stage: 'package',
+    type: 'json',
+    default: {
+      fileMatch: ['^build.gradle$'],
```

Comment:
Are build.gradle files only ever found in the root of a repo, or should we detect them within subdirectories too?